CHANGELOG.md

```diff
@@ -28,7 +28,6 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Breaking changes
 
--   If using auth wrapper without `requireAuth=false`, and the user signs out, then the user is automatically redirected to /auth
 -   The components override API has changed from `(DefaultComponent) => (props) => React.Element` to `({ DefaultComponent, …props }) => React.Element`.
```

Comment:
Need to add that on session expiry, the user will be redirected to the login page, unless the `onSessionExpired` callback is provided.